clusterDeployer.py

```diff
@@ -904,7 +905,7 @@ def boot_iso_x86(self, worker: NodeConfig, iso: str) -> None:
         lh = host.LocalHost()
         nfs = NFS(lh, self._cc.external_port)
 
-        bmc = host.bmc_from_host_name_or_ip(worker.node, worker.bmc_ip, worker.bmc_user, worker.bmc_password)
+        bmc = BMC.from_ip(worker.bmc_ip, worker.bmc_user, worker.bmc_password)
```

Comment:
why are we removing the ability to use worker.node? It is correct that we are making IP mandatory but why do we have to remove the abiilty to use .node? Could we make that optional instead instead of removing it altogether?

Reply:
I believe `worker.node` may not be needed since we don't need to resolve the worker's DNS name and add +1 to it's IPv4's address. Andrew please correct me if i am wrong.

Reply:
@vrindle bmc_from_host_name_or_ip wraps BMC.from_ip, and uses worker.node to determine the ip to hand to BMC.from_ip: https://github.com/bn222/cluster-deployment-automation/blob/main/host.py#L205.

This change is non-breaking provided the user specifies BMC_IP, which will now be required.